cutters/gpiocutter.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+# Copyright (c) 2013-2016 Intel, Inc.
+# Author Simo Kuusela <simo.kuusela@intel.com>
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; version 2 of the License
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+
+import os
+from aft.cutter import Cutter
+
+class GpioCutter(Cutter):
+    """
+    Class for handling Songle relay
+    """
+
+    def __init__(self, config):
+        pass
+
+    def connect(self):
+        """
+        Turns power on
+
+        Returns:
+            None
+        """
+        os.system("echo 1 > /sys/class/gpio/gpio60/value")
```

Comment:
We will probably need to use GPIOs also for emulating button presses (not necessarily with relays, for digital cases we should also support optical couplers), so I'd leave the name "relay" out from the base class. Another one can be derived for the specific relay/couple case - for example gpioswitch. The reason why I'd like to support both mechanical and optical switches is that they can be used to simulate different phenomenons. The mechanical switch will generate bounces, so it's good to test how the DUT can cope with it. The optical one, otoh, can be used to simulate other use cases (ex: end-of-line detection in a 3d printer or other similar contraption. Or optical encoder, to measure rotational speed).

But overall I agree that this should be improved.

However, if it works, I would rather merge this and then refine it in a follow up patch.